src/gcp/cloudfunctionsv2.ts

```diff
@@ -19,7 +19,7 @@ import {
   HASH_LABEL,
 } from "../functions/constants";
 
-export const API_VERSION = "v2alpha";
+export const API_VERSION = "v2";
```

Comment:
Only `v2` is introducing this change, right?

Otherwise all old CLI installs will break

Reply:
Yes old CLIs would display double entries. I don't think we have other choice though do we?

Reply:
Oh yeah, I'm not sure there's another way out of this then